### ISSUE TITLE
Use calculuswhiz/Assembly-Syntax-Definition for Unix Assembly syntax highlighting

### ISSUE THREAD

**Issue by tbodt** [github/linguist#4091]
Nessphoro/sublimeassembly is for NASM, which has a very different syntax from GAS (GNU assembler). The result looks pretty terrible.

**Comment by pchaigno** [comment]
Was there an issue with the template for issues?

**Comment by tbodt** [comment]
No, but it seemed more applicable to languages not being recognized correctly. In this case, the language is recognized correctly, but the syntax highlighting is wrong.

**Comment by pchaigno** [comment]
You can also review [the contribution guidelines](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#changing-the-source-of-a-syntax-highlighting-grammar) if you want to change the grammar.

**Comment by tbodt** [comment]
I did read the troubleshooting docs, but I didn't read the contribution guidelines. Sorry about that. I'll send a PR instead.
